CdCl_xfoil.m

```diff
@@ -0,0 +1,120 @@
+% |Author(s)   : Palma Caputo - Lorenzo Frascino                                                 |
+% |              University of Naples Federico II.                                               |
+% |Version     : 1.0                                                                             |
+% |Date        : 03/01/2020                                                                      |
+% |Modified    : 03/01/2020                                                                      |
+% |Description : evaluation of the Cd(Cl) polar by using the software xfoil                      |     
+% |Reference   : xfoil user's Guide                                                              |
+% |              <https://web.mit.edu/drela/Public/web/xfoil/xfoil_doc.txt>                      |                                                         
+% |Input       : (NACA) = NACA airfoil                                                           |
+% |              (ext_AF) = flag if airfoil geometry must be imported from external file         |  
+% |              (numPanel) = numver of panels for the geometry                                  |
+% |              (Re_number) = Reynolds number                                                   |
+% |              (iter) = number of iterations                                                   |
+% |              (FirstAlfa) = first value of the angle of attack                                |
+% |              (LastAlfa) = last value of the angle of attack                                  |
+% |              (DeltaAlfa) = pace between the angles of attack                                 |
+% |Output      : (Cl) = lift coefficient vector                                                  |
+% |              (Cd) = drag coefficent vector                                                   |
+% |Note        : The function must be in the same directory of xfoil.exe!                        |
+% ==============================================================================================
+function [Cl, Cd] = CdCl_xfoil(NACA, ext_AF, numPanel, Re_number, iter, FirstAlfa, LastAlfa, DeltaAlfa)
+
+Alfa_vec   = str2num(FirstAlfa):str2num(DeltaAlfa):str2num(LastAlfa);
+saveGeometry = 'Airfoil_geometry.txt';  % Create .txt file to save airfoil coordinates
+savePolar  = 'Polar.txt';               % Create .txt file to save the polar
+
+
+% Delete files if they exist
+if (exist(saveGeometry,'file'))
+    delete(saveGeometry);
+end
+
+if (exist(savePolar,'file'))
+    delete(savePolar);
+end
+
+
+%% WRITING XFOIL COMMANDS
+% Create the airfoil
+f_input = fopen('xfoil_input.txt','w');            % Create input file for xfoil 
+
+if ext_AF == 0
+    fprintf(f_input,'y\n');
+    fprintf(f_input,['naca ' NACA '\n']);
+end
+if ext_AF == 1
+    fprintf(f_input,'y\n');
+    fprintf(f_input,'load\n');
+    fprintf(f_input, 'my_airfoil.txt\n');          % .txt file with airfoil coordinates
+end

```

Comment:
See the previous comment.
Using one input you can check if it is an integer or a string.
In case it is an integer it should be interpreted as NACA, otherwise it should be interpreted as the name of the text file containing the airfoil coordinates.
To do this you can use the `isnumeric` function.